### ISSUE TITLE
which base model to use?

### ISSUE THREAD

**Issue by kkishore9891** [Peterande/D-FINE#79]
Hello, first of all thank you so much for your work! I can't thank you enough for your research.
I have a question for you. I am planning to train the D-Fine model on a custom proprietary dataset with the following metrics:

TRAIN Directory:
  Total images: 2825
  Total bounding boxes: 4189
    Class: 0, Count: 522
    Class: 1, Count: 810
    Class: 2, Count: 364
    Class: 3, Count: 820
    Class: 4, Count: 1583
    Class: 5, Count: 90

TEST Directory:
  Total images: 362
  Total bounding boxes: 506
    Class: 0, Count: 72
    Class: 1, Count: 113
    Class: 2, Count: 35
    Class: 3, Count: 79
    Class: 4, Count: 197
    Class: 5, Count: 10

VAL Directory:
  Total images: 533
  Total bounding boxes: 755
    Class: 0, Count: 98
    Class: 1, Count: 136
    Class: 2, Count: 34
    Class: 3, Count: 160
    Class: 4, Count: 308
    Class: 5, Count: 19

Since this is not a data rich dataset, I would like to know which base model to use as the starting point for training the model to have the least over fitting. Let us say we don't have more images or boxes yet, but we hope to soon increase the scale of our data by a lot. Thank you for the support.

**Comment by Peterande** [comment]
Given the small size of the current dataset, you may first try using D-FINE-S without pre-trained weights.

export model=s
CUDA_VISIBLE_DEVICES=0,1,2,3 torchrun --master_port=7777 --nproc_per_node=4 train.py -c configs/dfine/custom/dfine_hgnetv2_${model}_custom.yml --use-amp --seed=0

**Comment by kkishore9891** [comment]
Thank you for the support.

**Comment by kkishore9891** [comment]
Solved.
